### ISSUE TITLE
[Feature Request] Track combat movement history only for current turn

### ISSUE THREAD

**Issue by nschoenwald** [caewok/fvtt-elevation-ruler#232]
The module tracks movement history for the whole round. However, in DnD5e, when a token moves during another creature's turn, that is often because it is moved by force (pushed or dragged or something like that).

Only tracking the movement history during the current turn would much more often represent the movement one actually needs to track - the movement speed during a token's own turn.

**Comment by caewok** [comment]
In v12, you can clear the combatant movement with the button in the combat tracker. This seems like a simpler solution rather than doing something special just for dnd5e.

**Comment by nschoenwald** [comment]
> In v12, you can clear the combatant movement with the button in the combat tracker. This seems like a simpler solution rather than doing something special just for dnd5e.

I don't think it would be special for dnd5e. Are there a lot of systems that need you to track the movement across the whole round?
From my experience, most systems only need you to track the movement for the current combatant's turn to ensure they do not exceed their movement limit. If they get moved outside of their own turn, that does not count against their movement limit.
I believe only tracking for the current turn, not the combat round, would be a better default behavior for the vast majority of systems.
